services/construction.go

```diff
@@ -43,8 +43,8 @@ const DefaultGas transaction.Gas = 10000
 
 // UnsignedTransaction is a transaction with the account that would sign it.
 type UnsignedTransaction struct {
-	Tx     transaction.Transaction `json:"tx"`
-	Signer string                  `json:"signer"`
+	Tx     cbor.RawMessage `json:"tx"`
```

Comment:
:information_source: this change is because previously, in several places, unmarshal UnsignedTransaction and then marshal the .Tx. now those places access .Tx directly and some other places further unmarshal .Tx